package/efs/rancher-efs

```diff
@@ -106,6 +106,15 @@ create() {
 
     get_meta_data
 
+    # Check if an EFS volume with the same creation token exists
+    local existing_fs
+    existing_fs=`aws efs describe-file-systems --region ${EC2_REGION} --creation-token ${OPTS[name]} | grep -c ${OPTS[name]}`
+    
```

Comment:
I am checking the existence of file system before even making the create call.

This problem is encountered if we deleted the current stack that accessed the volume and now create a new stack that is using the volume. Rancher still goes ahead with creating a new volume which fails since EFS with that --creation-token already exists.